setup.py

```diff
@@ -112,11 +112,11 @@ def run(self):
             check=True,
         )
         subprocess.run(
-            ["cmake", "--build", build_dir],
+            ["cmake", "--build", build_dir, "--parallel"],
             check=True,
         )
```

Comment:
Let's remove this whole thing and let the one below take care of the build and the install.

Reply:
Okk